american-physiological-society.csl

```diff
@@ -212,11 +212,15 @@
           </choose>
         </else-if>
         <else>
-          <text variable="container-title" suffix=" " form="short" font-style="italic"/>
-          <text variable="volume"/>
-          <text variable="publisher" suffix="."/>
-          <text variable="page" prefix=": " suffix=", "/>
-          <text macro="date" prefix=" " suffix="."/>
+          <group delimiter=" ">
```

Comment:
No problem. I can add that group. I think we are here in the last `else` case, i.e. we don't even know what strange itemType we are dealing. Thus, maybe there will also be cases as you described.